### PR TITLE
Reduces friction for CUJ "I want to know about the supported manifest keys"

### LINKED ISSUE #4254: [CRX CUJ] I want to know about the supported manifest.json keys
[Friction Log](https://docs.google.com/document/d/1VP-l40QJnzzQTv13ZFqrTwJhRciw-fdQ1U_otlhDBpk/edit?usp=sharing)

### PR DESCRIPTION
**Pull request by AmySteam**
Fixes #4254. #163. #1774. #1859 

Changes proposed in this pull request:

- Remove keys for internal use only 
- Add links to missing docs ([PR#4293](https://github.com/GoogleChrome/developer.chrome.com/pull/4293), [PR#4289](https://github.com/GoogleChrome/developer.chrome.com/pull/4289), #4291, #3837)
- Add code samples to all manifest key pages
- Standardize format for all keys: "key name and link": "key type"
  - For example: "[action](https://developer.chrome.com/docs/extensions/reference/action/)": {...},
- Add manifest code samples from the Getting Started guide tutorials so that developers can copy/paste them as examples in their projects. 

**Keys ready for review**
- [ ] [manifest_version](https://deploy-preview-4255--developer-chrome-com.netlify.app/docs/extensions/mv3/manifest/manifest_version/)
- [ ] [sandbox](https://deploy-preview-4255--developer-chrome-com.netlify.app/docs/extensions/mv3/manifest/sandbox/)
- [ ] [version](https://deploy-preview-4255--developer-chrome-com.netlify.app/docs/extensions/mv3/manifest/version/)
- [ ] [description](https://deploy-preview-4255--developer-chrome-com.netlify.app/docs/extensions/mv3/manifest/description/)
- [ ] [externally_connectable](https://deploy-preview-4255--developer-chrome-com.netlify.app/docs/extensions/mv3/manifest/externally_connectable/)
- [ ] [default_locale/](https://deploy-preview-4255--developer-chrome-com.netlify.app/docs/extensions/mv3/manifest/default_locale/)
- [ ] [key](https://deploy-preview-4255--developer-chrome-com.netlify.app/docs/extensions/mv3/manifest/key)
- [ ] [optional host permissions](https://deploy-preview-4255--developer-chrome-com.netlify.app/docs/extensions/reference/permissions/#step-2-declare-optional-permissions-in-the-manifest)

**Staged links**
[Manifest keys](https://deploy-preview-4255--developer-chrome-com.netlify.app/en/docs/extensions/mv3/manifest/)

### CODE REVIEW

**Comment by jpmedley** [comment]
@AmySteam Did you find anything in here that should be mentioned in the upgrade materials?

**Comment by AmySteam** [comment]
@jpmedley Not really. Mostly cleaned up broken links and removed mentions of Chrome apps.

**Comment by jpmedley** [comment]
@AmySteam, if this is ready for fact checking, please assign it to @dotproto.

Thanks

File: site/en/docs/extensions/mv3/manifest/key/index.md [hunk 1]
```diff
@@ -2,22 +2,18 @@
...
 title: "Manifest - Key"
 date: 2013-05-12
-updated: 2018-04-26
+updated: 2022-11-29
 description: Reference documentation for the key property of manifest.json.
 ---
 
-This value can be used to control the unique ID of an extension, app, or theme when it is loaded
-during development.
+This value can be used to maintain the unique ID of an extension, or theme when it is loaded
```
Comment (jpmedley) [inline site/en/docs/extensions/mv3/manifest/key/index.md:9]:
```suggestion
This value maintains the unique ID of an extension, or theme when it is loaded
```

Comment (jpmedley) [reply site/en/docs/extensions/mv3/manifest/key/index.md:9]: Omit needless words.

Would 'stores' be a better verb?

**Review by jpmedley — CHANGES_REQUESTED** [review]

**Comment by AmySteam** [comment]
Hey @dotproto, this is ready for a technical review; I assigned it to you 🙂

**Review by jpmedley — APPROVED** [review]

**Pull request merged**